src/gc.c

```diff
@@ -1901,7 +1901,7 @@ excstack: {
         size_t i = stackitr->i;
         while (itr > 0) {
             size_t bt_size = jl_exc_stack_bt_size(exc_stack, itr);
-            uint64_t *bt_data = jl_exc_stack_bt_data(exc_stack, itr);
+            uintptr_t *bt_data = jl_exc_stack_bt_data(exc_stack, itr);
```

Comment:
Hah, this was a hilariously stupid bug which cost a lot of time due to randomly busted GC on 32 bit, I don't know what I was thinking using `uint64_t` here :-/

It would have been caught early if we were able to use `-Werror` on CI builds but I suspect that is quite disruptive in other ways.

Reply:
> It would have been caught early if we were able to use -Werror on CI builds but I suspect that is quite disruptive in other ways.

Please, we should absolutely do this. We've had a creeping increase of compiler warnings in the past month or so.